git-hub

```diff
@@ -293,7 +293,7 @@ def editor(help_msg, msg=None):
 	with file(fname, 'w') as f:
 		f.write(msg or '')
 		f.write(help_msg)
-	status = subprocess.call([prog, fname])
+	status = subprocess.call(['sh', '-c', prog + ' "$@"', prog, fname])
```

Comment:
I think this is equivalent to:
```py
subprocess.call([prog + ' "$@"', prog, fname], shell=True)
```
But in this case you are leaving which shell to use to Python (which defaults to `/bin/sh` anyway).